common/middleware/local_node.py

```diff
@@ -17,9 +17,19 @@ def process_request(self, request):
             if host[-1] != "/":
                 host += "/"
 
+            service = host + "service/"
+
             if len(nodes) == 0:
-                node = Node(name="Local", website_url=host, service_url=host+"service/", local=True)
+                node = Node(name="Local", website_url=host, service_url=service, local=True)
+                node.save()
+            elif len(nodes) == 1:
+                node = nodes[0]
+                node.host = host
+                node.service = service
```

Comment:
Haha and I was thinking about suggesting that you name `website_url` to `host` when you submitted your initial pull request for the `Node` model. 😹